src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6337,6 +6337,18 @@ public sealed partial class AsyncStateMachineAttribute : System.Runtime.Compiler
     {
         public AsyncStateMachineAttribute(System.Type stateMachineType) : base (default(System.Type)) { }
     }
+    public partial struct AsyncValueTaskMethodBuilder
```

Comment:
Maybe a stupid question, but why are all these structs partial?

Reply:
I don't know; the ref generator does it this way.  I presume to make it easier to have all of the core code in one file, and then have refs for different platforms that add additional APIs.